### DISCUSSION TITLE
[FEATURE REQUEST] Specify refresh interval at launch or in profile

### DISCUSSION THREAD

**Discussion by 564684684165186** [aristocratos/btop#376] [Ideas]
First, awesome application, love it.

I'd love to be able to set refresh time in a profile, I'm happy with my layout and want to be able to change to a daily graph from a two second graph without having to go through the menu, being able to do this vi profile is the easiest way I can think to do it.
Otherwise, being able to specify the refresh interval with a flag would work. OR being able to point to different config files on launch would work too, then I can have multiple btop.conf with different intervals, and restart the program to change them.

**Comment by mrghosti3** [comment]
Isn't the refresh rate always visible at the top right corner? It even shows -/+ keys that are used to change the refresh rate.

**Reply by mrghosti3** [reply]
Never mind. Misunderstood the feature request.
